src/connector/src/sink/remote.rs

```diff
@@ -145,7 +151,15 @@ impl<R: RemoteSinkTrait> Sink for RemoteSink<R> {
     }
 
     async fn new_log_sinker(&self, writer_param: SinkWriterParam) -> Result<Self::LogSinker> {
-        RemoteLogSinker::new(self.param.clone(), writer_param).await
+        RemoteLogSinker::new(
```

Comment:
All new parameters are already in `self.param.clone()`. Why pass them separatedly?